tests/test_import_interceptor.py

```diff
@@ -1,7 +0,0 @@
-import pytest
-
-
-def test_prefect_1_import_warning():
-    with pytest.raises(ImportError):
-        with pytest.warns(UserWarning, match="Attempted import of 'prefect.Client"):
-            from prefect import Client  # noqa
```

Comment:
This test is testing for Prefect 1 import warnings which have been remove, so I don't think this test is needed any longer.